blocks/header/header.css

```diff
@@ -53,125 +41,49 @@ header nav p {
   line-height: 1;
 }
 
-header nav a:any-link {
-  color: currentcolor;
-}
-
-/* hamburger */
-header nav .nav-hamburger {
-  grid-area: hamburger;
-  height: 22px;
-  display: flex;
+/* brand */
+header .nav-brand {
+  display: inline-flex;
   align-items: center;
+  gap: 16px;
+  position: relative;
+  align-self: stretch;
+  flex: 0 0 auto;
 }
 
-header nav .nav-hamburger button {
-  height: 22px;
-  margin: 0;
-  border: 0;
-  border-radius: 0;
-  padding: 0;
-  background-color: var(--background-color);
-  color: inherit;
-  overflow: initial;
-  text-overflow: initial;
-  white-space: initial;
+header .nav-brand .logo {
+  position: relative;
+  width: 4rem;
 }
 
-header nav .nav-hamburger-icon,
-header nav .nav-hamburger-icon::before,
-header nav .nav-hamburger-icon::after {
-  box-sizing: border-box;
-  display: block;
+header .nav-brand .group {
   position: relative;
-  width: 20px;
+  height: 3.66vh;
```

Comment:
The height should be fixed, as otherwise the icon becomes invisible if one reduces the screen height. Example: Open the page in a browser, then activate the dev tools (to show beneath the window). As the window height is reduced, the icon becomes less visible.